addons/mail/static/src/components/discuss_sidebar/discuss_sidebar.xml

```diff
@@ -12,69 +12,123 @@
                 </t>
             </div>
             <hr class="o_DiscussSidebar_separator"/>
-            <t t-if="env.models['mail.thread'].all(thread => thread.isPinned and thread.model === 'mail.channel').length > 19">
-                <input class="o_DiscussSidebar_quickSearch" t-on-input="_onInputQuickSearch" placeholder="Quick search..." t-ref="quickSearchInput" t-esc="discuss.sidebarQuickSearchValue"/>
-            </t>
-            <div class="o_DiscussSidebar_group o_DiscussSidebar_groupChannel">
-                <div class="o_DiscussSidebar_groupHeader">
-                    <div class="o_DiscussSidebar_groupHeaderItem o_DiscussSidebar_groupTitle o-clickable" t-on-click="_onClickChannelTitle">
-                        Channels
+            <div class="o_DiscussSidebar_threadContainer">
```

Comment:
Too much diff, do you really need to add this? But sidebar will be completely changed by QHO anyway.